### ISSUE TITLE
ServiceSubscriberInterface missing in SessionListener

### ISSUE THREAD

**Issue by jeserkin** [silexphp/Silex#1496]
So I have usual code
```
use Silex\Provider\SessionServiceProvider;

$app = new Silex\Application();
$app->register( new SessionServiceProvider() );
```

And once it tries to run this code I get following error:
```
[SYSLOG] php-fpm[31]: [WARNING] [pool www] child 59 said into stderr: "NOTICE: PHP message: PHP Fatal error:  Interface 'Symfony\Component\DependencyInjection\ServiceSubscriberInterface' not found in /app/backend/vendor/symfony/http-kernel/EventListener/SessionListener.php on line 26"
[SYSLOG] php-fpm[31]: [WARNING] [pool www] child 59 said into stderr: "NOTICE: PHP message: PHP Stack trace:"
[SYSLOG] php-fpm[31]: [WARNING] [pool www] child 59 said into stderr: "NOTICE: PHP message: PHP   1. {main}() /app/backend/app/index.php:0"
[SYSLOG] php-fpm[31]: [WARNING] [pool www] child 59 said into stderr: "NOTICE: PHP message: PHP   2. Silex\Application->run() /app/backend/app/index.php:8"
[SYSLOG] php-fpm[31]: [WARNING] [pool www] child 59 said into stderr: "NOTICE: PHP message: PHP   3. Silex\Application->handle() /app/backend/vendor/silex/silex/src/Silex/Application.php:477"
[SYSLOG] php-fpm[31]: [WARNING] [pool www] child 59 said into stderr: "NOTICE: PHP message: PHP   4. Silex\Application->boot() /app/backend/vendor/silex/silex/src/Silex/Application.php:491"
[SYSLOG] php-fpm[31]: [WARNING] [pool www] child 59 said into stderr: "NOTICE: PHP message: PHP   5. Silex\Provider\SessionServiceProvider->subscribe() /app/backend/vendor/silex/silex/src/Silex/Application.php:111"
[SYSLOG] php-fpm[31]: [WARNING] [pool www] child 59 said into stderr: "NOTICE: PHP message: PHP   6. Pimple\Container->offsetGet() /app/backend/vendor/silex/silex/src/Silex/Application.php:80"
[SYSLOG] php-fpm[31]: [WARNING] [pool www] child 59 said into stderr: "NOTICE: PHP message: PHP   7. Silex\Provider\SessionServiceProvider->Silex\Provider\{closure}() /app/backend/vendor/pimple/pimple/src/Pimple/Container.php:113"
[SYSLOG] php-fpm[31]: [WARNING] [pool www] child 59 said into stderr: "NOTICE: PHP message: PHP   8. spl_autoload_call() /app/backend/vendor/pimple/pimple/src/Pimple/Container.php:60"
[SYSLOG] php-fpm[31]: [WARNING] [pool www] child 59 said into stderr: "NOTICE: PHP message: PHP   9. Composer\Autoload\ClassLoader->loadClass() /app/backend/vendor/pimple/pimple/src/Pimple/Container.php:60"
[SYSLOG] php-fpm[31]: [WARNING] [pool www] child 59 said into stderr: "NOTICE: PHP message: PHP  10. Composer\Autoload\includeFile() /app/backend/vendor/composer/ClassLoader.php:322"
[SYSLOG] php-fpm[31]: [WARNING] [pool www] child 59 said into stderr: "NOTICE: PHP message: PHP  11. include() /app/backend/vendor/composer/ClassLoader.php:444"
[SYSLOG] php-fpm[31]: [WARNING] [pool www] child 59 said into stderr: "NOTICE: PHP message: PHP  12. spl_autoload_call() /app/backend/vendor/composer/ClassLoader.php:23"
[SYSLOG] php-fpm[31]: [WARNING] [pool www] child 59 said into stderr: "NOTICE: PHP message: PHP  13. Composer\Autoload\ClassLoader->loadClass() /app/backend/vendor/composer/ClassLoader.php:23"
[SYSLOG] php-fpm[31]: [WARNING] [pool www] child 59 said into stderr: "NOTICE: PHP message: PHP  14. Composer\Autoload\includeFile() /app/backend/vendor/composer/ClassLoader.php:322"
[SYSLOG] php-fpm[31]: [WARNING] [pool www] child 59 said into stderr: "NOTICE: PHP message: PHP  15. include() /app/backend/vendor/composer/ClassLoader.php:444"
```

I've installed `"silex/silex": "2.0.3"` and 2.0.4 had same code in that area.

**Comment by GromNaN** [comment]
Are you using symfony master branch ? It is related to symfony/symfony#21708: https://github.com/symfony/symfony/pull/21708/files#diff-68df7c6efefbcf008cd3c0efb8f69b64
Since [Pimple is not compatible with PSR-11](https://github.com/silexphp/Pimple/pull/216) and the Symfony DI component will never be required by Silex, the solution is to create a subclass of [AbstractSessionListener](https://github.com/symfony/symfony/blob/25d17345c022884aafb2263766974dc8ca5bfebb/src/Symfony/Component/HttpKernel/EventListener/AbstractSessionListener.php) that is appearing in Symfony 3.3.

**Comment by jeserkin** [comment]
Will it be resolved on silex (dependency) level?

**Comment by GromNaN** [comment]
Silex already have the classes. I've submitted a PR to fix symfony code: symfony/symfony#22171

**Comment by jeserkin** [comment]
I haven't specified symfony separately. Just Silex. So it seems it comes from there through dependencies. Maybe, the issue is, that I have in composer.json `"minimum-stability": "dev-master"`?
